packages/lib/responseNote/cache.ts

```diff
@@ -0,0 +1,10 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateProps {
+  id?: string;
+}
+
+export const responseNoteCache = {
```

Comment:
This caching strategy seems wrong 😉
I think if we would like to do it correctly we should also have a `getResonseNotes(responseId: string)` method that is called within the response service to get the responseNotes here, instead of making the prisma select database call. What do you think?

Reply:
Same would be true for the tags.

Reply:
yeah that makes sense. so instead of querying the db directly for the responseNotes in the `response` service, we have a function in the responseNote service that is cached and called from the `response` service.

but I don't think I understand here what you mean by the tags.

Reply:
@mattinannt Updated, please can you take a look?